resources/web/data/text.js

```diff
@@ -1,5 +1,5 @@
 var LangText={
- "en":{
+ "en_EN":{
```

Comment:
Curious, does this change improve the compatibility or fix a bug?

Reply:
To be honest I don't know ... I have change this Langage variable because for the French Translation if you don't use "fr_FR" then the translatation is not loaded on the intial web page . It was already the case on the chinese translation and I have made the test for the German langage. Now concerning the english one , as the original langage is alreday in english it should not change a lot of things .. I just assumed that the tag should have this shape like for the other languages.

Reply:
I was wrong .. for the english you must leave "en"  and not "en_EN"  !